src/index.js

```diff
@@ -142,6 +142,16 @@ fireEvent.select = (node, init) => {
   fireEvent.keyUp(node, init)
 }
 
+// if we're running in a test runner that supports afterEach
+// then we'll automatically run cleanup afterEach test
+// this ensures that tests run in isolation from each other
+if (typeof afterEach === 'function' && !process.env.RTL_SKIP_CLEANUP) {
```

Comment:
We should guard against process and process.env not being defined for people running their tests in actual browsers.